juice/capnp/juice.capnp

```diff
@@ -62,11 +62,13 @@ struct ConvolutionConfig {
 }
 
 struct RnnConfig {
-	outputSize @0 :UInt64;
-	cellSize @1 :UInt64;
-	hiddenSize @2 :UInt64;
-	numLayers @3 :UInt64;
-	rnnType @4 :Text;
+	hiddenSize @0 :UInt64;
```

Comment:
changing the juice.capnp should always modify the id at the very top too, IIRC you can use the `capnp` tool to do that.